StarRocks Logging and Related Configurations (#267)
StarRocks/starrocks-kubernetes-operator: doc/logging_and_related_configurations_howto.md

# StarRocks Logging and Related Configurations

> StarRocks comprises three components: FE, BE, and CN. Among them, FE is essential. This document introduces the logging

Reviewer: ```suggestion
StarRocks is comprised of three components: FE, BE, and CN. Among them, FE is essential. This document describes the logging
```

and associated configurations for StarRocks components, including:

1. Location of log storage;
2. Default storage volume;
3. Persisting logs;
4. Logging to the console;
5. Collecting logs into Datadog.

## 1. Location of Log Storage

1. The logs of the FE component are stored in: `/opt/starrocks/fe/log`, with key logs including: `fe.out`, `fe.log`, `fe.warn.log`.
2. The BE component's logs are located in: `/opt/starrocks/be/log`, key logs being: `be.out`, `be.INFO`, `be.WARNING`.

> 3. CN component logs can be found in: `/opt/starrocks/cn/log`, with significant logs such as: `cn.out`, `cn.INFO`, `cn.WARNING`.

Reviewer: use same description and terms for all components.

## 2. Default Storage Volume

> By default, all components use the `emptyDir` storage volume. One inherent problem is that once a Pod restarts, we can't access logs from before the restart, which obviously complicates troubleshooting. To address this, we can adopt one of two approaches:

Reviewer: ```suggestion
By default, all components use the `emptyDir` storage volume. One inherent problem is that once a Pod restarts, logs before the restart will not be accessible any more, which obviously complicates troubleshooting. To address this, one of two approaches can be adopted:
```
